### PR TITLE
feat: add is_online_mode() API in Python SDK

### PR DESCRIPTION
**Pull request by HuilinWu2**
We can now see the current execute mode by `cursor.is_online_mode()`.

### CODE REVIEW

File: python/openmldb/sdk/sdk.py [hunk 1]
```diff
@@ -127,6 +127,11 @@ def execute_sql(self, db, sql):
             return False, status.msg
         else:
             return True, rs
+        
+    def is_Online_Mode(self):
```
Comment (tobegit3hub) [inline python/openmldb/sdk/sdk.py]: Rename the function name such as `isOnlineMode()`.

Comment (HuilinWu2) [reply python/openmldb/sdk/sdk.py]: updated.

**Review by dl239 — APPROVED** [review]

**Review by tobegit3hub — APPROVED** [review]
LGTM

**Comment by tobegit3hub** [comment]
Make execution mode as offline by default and add test cases for offline mode.

**Comment by HuilinWu2** [comment]
[Improve functionality of Python SDK](https://trello.com/c/xlXZPfTf/20-improve-functionality-of-python-sdk)

**Pull request merged**